docs/kubernetes.md

```diff
@@ -32,6 +32,34 @@ In the image above, you can see the following parts:
 
 All VMs are in the same private VNET and are fully accessible to each other.
 
+
+## Optional: Enable network policy enforcement using Calico

```

Comment:
I think it should be split in two.  Most of this stays here as the "how to use it" part of the walk through.  But the `azuredeploy.parameters.json` oriented bit moves up to be just another numbered step in the deployment section near the top of the doc.